test_process_schemaless.py

```diff
@@ -1,7 +1,32 @@
 # Lint as: python3
+from datetime import datetime
+from collections import defaultdict
 from collections import namedtuple
 
+from process_schemaless import extract_freshness
 from process_schemaless import is_seen_id
+from process.types import four_level_dict
+
+
+def test_extract_freshness():
+    data = defaultdict(lambda: four_level_dict())
+
+    newer = datetime.fromisoformat('2020-01-01')
+
+    data['uuid1']['ppts']['PRJ']['date_opened']['value'] = '01/01/2000'
+    data['uuid2']['ppts']['PRJ']['date_opened']['value'] = '01/01/2010'
+    data['uuid3']['ppts']['PRJ']['date_opened']['value'] = '01/01/2020'
+
+    # ignored
```

Comment:
Nit: Let's put that this is ignored because the field is an arbitrary value (similar to comment below), I couldn't immediately see when I was glancing through the code